sysutils/fx/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        antonmedv fx 18.0.1
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         Command-line tool and terminal JSON viewer
+long_description    ${description}
+
+checksums           rmd160  86d17a60e47f88a334292566ae584af06c08343e \
+                    sha256  a0e6bf4bac205a5b04b9553d47692a6b80b9fc34e2b679c8472b70686ca78cd5 \
+                    size    11106322
+
+master_sites        ${github.homepage}/releases/download/${version}
+distname            ${name}-macos
+
+extract.mkdir       yes
```

Comment:
GitHub portgroup detects fx-18.0.1.tar.gz, but we need fx-macos.zip here.

Reply:
Actually only master_sites is not necessary here.

Reply:
> GitHub portgroup detects fx-18.0.1.tar.gz, but we need fx-macos.zip here.

That's because you have the line `use_zip             yes` which needs to be removed too.

Reply:
The GitHub portgroup pretty much always selects the correct distfile name. You just need to let it do its magic.

Reply:
Well, I've tried to remove those 4 line - it doesn't work, GitHub portgroup cannot guess correct distfile.

Reply:
Removed `master_sites`.

Reply:
You might need to specify a release download then. 

`github.tarball_from  releases
`

Reply:
Added.

Reply:
@mf2k Is it OK now? Can we land this pr?